src/SDKs/_metadata/apimanagement_resource-manager.txt

```diff
@@ -1,10 +0,0 @@
-2017-08-01 15:52:08 UTC
```

Comment:
Please don't remove this.  This file is used to track the metadata used to create your package.  When you run generate.cmd, it should replace this file for you